collections/README.md

```diff
@@ -32,115 +28,116 @@ The following Link relations shall exist in the Landing Page (root).
 
-Additionally, `child` relations may exist to individual catalogs and collections.
+Additionally, these relations may exist:
+
 | **rel**        | **href**             | **From**       | **Description**                                                                                                                                                         |
 | -------------- | -------------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `child`        | various              | STAC Core      | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects |
 
-| `child`        | various              | STAC Core      | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects                                                                |
+`child` relations are useful for supporting browsing a STAC API as if it were a static catalog.
 
-The following Link relations should exist in the `/collections` endpoint response.
+The following Link relations shall exist in the `/collections` endpoint response.
 
 | **rel** | **href**       | **From**  | **Description** |
 | ------- | -------------- | --------- | --------------- |
 | `root`  | `/`            | STAC Core | The root URI    |
 | `self`  | `/collections` | OAFeat    | Self reference  |
 
-The following Link relations should exist in the `/collections/{collectionId}` endpoint response.
+The following Link relations shall exist in the Collection object returned from the `/collections/{collectionId}` endpoint.
 
 | **rel**  | **href**                            | **From**  | **Description**                            |
 | -------- | ----------------------------------- | --------- | ------------------------------------------ |
 | `root`   | `/`                                 | STAC Core | The root URI                               |
 | `parent` | `/`                                 | OAFeat    | Parent reference, usually the root Catalog |
 | `self`   | `/collections/{collectionId}`       | OAFeat    | Self reference                             |
-| `items`  | `/collections/{collectionId}/items` | OAFeat    | Items within Collection                    |
+
+Additionally, these relations may exist for the `/collections/{collectionId}` endpoint:
+
+| **rel**  | **href**                            | **From**  | **Description**                            |
+| -------- | ----------------------------------- | --------- | ------------------------------------------ |
+| `canonical`        | various              | STAC Core      | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects |
```

Comment:
I don't really understand what this means?!

Reply:
I tried to clarify. This was one of @cholmes's ideas -- I think the main use cases is federated catalogs, where the canonical url (to another catalog in another place) is different from the URL you used to retrieve the catalog.

Reply:
I think it might be easier to understand if we align with the RFC6596 description of canonical:

>    The canonical link relation specifies the preferred IRI from
   resources with duplicative content.  Common implementations of the
   canonical link relation are to specify the preferred version of an
   IRI [...]

Even in the revised version there's a lot of "canonical" in the text, which feels like a recursive definition.

Also, having 
> The child STAC Catalogs & Collections. 

at the beginning of the description is a bit confusing as it conflicts with the "child" relation, which is also used for "The child STAC Catalogs & Collections."

Maybe we should go with something like:
> Provides the preferred paths to get to STAC Collection and Item objects, if they differ from the URL that was used to retrieve the STAC object and thus duplicate other content. This can be useful in federated catalogs that present metadata that has a different location than the source metadata.

Reply:
sounds good to me, updated.